LCM/dsc/engine/ca/CAInfrastructure/WebPullClient.c

```diff
@@ -1281,7 +1226,7 @@ MI_Result SetGeneralCurlOptions(CURL* curl,
 {
     CURLcode res;
     curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, 0L);
-    curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
+    curl_easy_setopt(curl, CURLOPT_VERBOSE, 0L);
```

Comment:
>0L [](start = 44, length = 2)

?

Reply:
it generated a lot of verbose messages. python wrappers treat that as error.

---
In reply to: [276398740](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/553#discussion_r276398740) [](ancestors = 276398740)